src/components/AnonymousReportFooter.tsx

```diff
@@ -50,7 +46,7 @@ function AnonymousReportFooter({isSmallSizeLayout = false, report, policy}: Anon
                     <Button
                         success
                         text={translate('common.signIn')}
-                        onPress={() => Session.signOutAndRedirectToSignIn()}
+                        onPress={() => signOutAndRedirectToSignIn()}
```

Comment:
We should use a stable reference here:

```suggestion
                        onPress={signOutAndRedirectToSignIn}
```

Reply:
Ah, then this should do the trick:

```
```
```suggestion
                        onPress={signOutAndRedirectToSignIn as () => void}
```